mayastor/src/bdev/nexus/nexus_io.rs

```diff
@@ -130,7 +155,7 @@ impl Bio {
         self.ctx_as_mut_ref().in_flight -= 1;
 
         if cfg!(debug_assertions) {
-            assert_ne!(self.ctx_as_mut_ref().in_flight, -1);
+            assert!(self.ctx_as_mut_ref().in_flight >= 0);
```

Comment:
This block can just be a debug_assert!().

Reply:
Ok, didn't know you could do that.